Test: this is msg from error_handling()
